documentation/src/main/asciidoc/user_guide/chapter-76-Simple_Cache.adoc

```diff
@@ -0,0 +1,47 @@
+== Simple Cache
+
+Traditional local caches use the same architecture as clustered caches - interceptor stack. That way a lot of the functionality can be reused. However, if the advanced features are not needed and the performance is more important, the interceptor stack can be stripped away and simple cache can be used.
```

Comment:
1) ok
2) really? Opening documentation, it is usually whole paragraph on one line. And I prefer using word wrapping in IDE (IDEA does that quite well once you map the to key shortcut) rather than rewrapping that manually when I need to edit that.